cmake/PybindWrap.cmake

```diff
@@ -55,39 +55,55 @@ function(
     set(GTWRAP_PATH_SEPARATOR ";")
   endif()
 
+  # Create a copy of interface_headers so we can freely manipulate it
+  set(interface_files ${interface_headers})
+
+  # Pop the main interface file so that interface_files has only submodules.
+  LIST(POP_FRONT interface_files main_interface)
```

Comment:
Just a picky detail: list is preferred in lowercase.

Reply:
Please nitpick as much as possible. Having you here as a reviewer is a great opportunity for me to learn. :slightly_smiling_face: